test/parallel/test-whatwg-url-origin.js

```diff
@@ -50,3 +51,12 @@ function runURLTests(urltests) {
 
 runURLOriginTests()
 /* eslint-enable */
+
+// Tests below are not from WPT.
+const tests = require(path.join(common.fixturesDir, 'url-tests-additional.js'));
```

Comment:
Should we add a test into the other URL tests as well to use `url-tests-addition`? We also can separate the patch from this PR :)